.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,18 @@
+name: Lighthouse CI
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [labeled] # Only run when a PR is labeled with the 'Seen-on-PROD' label set by prout
+jobs:
+  Lighthouse:
+    if: ${{ github.event.label.name == 'Seen-on-PROD' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Audit URLs using Lighthouse
+        uses: treosh/lighthouse-ci-action@v12
+        with:
+          urls: |
+            https://support.theguardian.com/uk/contribute
```

Comment:
We should have a think about other urls to add here. The subscribe ones for instance.